hw/ip/rom_ctrl/doc/checklist.md

```diff
@@ -143,9 +143,9 @@ Coverage      | [SIM_COVERAGE_MODEL_ADDED][]          | Done        |
 Code Quality  | [TB_LINT_SETUP][]                     | Done        |
 Integration   | [PRE_VERIFIED_SUB_MODULES_V1][]       | N/A         |
 Review        | [DESIGN_SPEC_REVIEWED][]              | Done        |
-Review        | [TESTPLAN_REVIEWED][]                 | Not Started |
+Review        | [TESTPLAN_REVIEWED][]                 | Done        |
```

Comment:
For anyone looking at this in the future, there was a V1 review meeting on Jan 7th. For OT members, the (rather boring) minutes are [here](https://docs.google.com/document/d/1YNTIqfv9iuUR7DbiiyJ5f71UQK5gwp_n8zfKcvfXXEo).